tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/im2latex.py

```diff
@@ -36,7 +36,7 @@ def read_vocab(vocab_path):
     """
     if vocab_path.suffix == '.pkl':
         with open(vocab_path, "rb") as f:
-            vocab_dict = pkl.load(f)
+            vocab_dict = pkl.load(f) # nosec
```

Comment:
Please add explanation to what you are aiming to suppress. Similar to https://github.com/openvinotoolkit/open_model_zoo/blob/9c9c3994367a6ac10a0d645c808dadcfcf99668a/tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/im2latex.py#L17

Reply:
Done